src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -467,5 +475,77 @@ private static string GetTempAssemblyName(AssemblyName parent, string ns)
         {
             return parent.Name + ".XmlSerializers" + (ns == null || ns.Length == 0 ? "" : "." + ns.GetHashCode());
         }
+
+        private static void ParseReferences(string value, Dictionary<string, string> dictionary)
+        {
+            List<string> list = new List<string>();
+            if (value.Length > 0)
```

Comment:
Use more descriptive name than list. Also you can use var as you know what the type is as you are new-ing up the instance on the same line.